doc/classes/SpriteFrames.xml

```diff
@@ -39,6 +39,14 @@
 				Removes all animations. An empty [code]default[/code] animation will be created.
 			</description>
 		</method>
+		<method name="duplicate_animation">
+			<return type="void" />
+			<param index="0" name="anim_from" type="StringName" />
+			<param index="1" name="anim_to" type="StringName" />
+			<description>
+				Duplicate all frames [param anim_from] to the [param anim_to].
```

Comment:
```suggestion
				Duplicate all frames from [param anim_from] to [param anim_to].
```

Reply:
I think the current description is misleading. It sounds like the animation has to already exist, while it doesn't need to.